openreview/tools.py

```diff
@@ -1454,6 +1454,33 @@ def get_neurips_profile_info(profile, n_years=None):
         'publications': publications
     }
 
+def get_sac_profile_info(profile, submission_venue_id):
```

Comment:
I would call it with a name related to the implementation because it can be used in other places. 

What about? `get_current_submissions_profile_info`